projects/observability/src/pages/api-trace-detail/api-trace-detail.page.module.ts

```diff
@@ -32,7 +33,8 @@ const ROUTE_CONFIG: TraceRoute[] = [
     LoadAsyncModule,
     FormattingModule,
     ButtonModule,
-    CopyShareableLinkToClipboardModule
+    CopyShareableLinkToClipboardModule,
+    DownloadJsonModule
```

Comment:
was the api trace detail page changed to add this? I only see the full trace detail

Reply:
Yeah, It is only for full trace page.

Reply:
in that case, should it be removed here?

Reply:
Yeah removed!!
Thanks for pointing that out.